src/components/DisplayQuates.js

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from 'react';
+
+const category = 'knowledge';
+const apiUrl = `https://api.api-ninjas.com/v1/quotes?category=${category}`;
+const apiKey = '8d3AEu5Zw1oN3/7ZcRczQw==Pttifja0qKZnwTuq';
+
+function Display() {
+  const [quote, setData] = useState([]);
+  const [hasError, setHasError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setIsLoading(true);
+      try {
+        const response = await fetch(apiUrl, {
+          headers: {
+            'X-Api-Key': apiKey,
+          },
+        });
+        const data = await response.json();
+        console.log(data);
```

Comment:
- [OPTIONAL] Kindly remove `console.log` from your codebase in order to keep it clean and professional.